### DISCUSSION TITLE
Insights > time series > custom time could be better

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#12945] [Archive]
https://user-images.githubusercontent.com/62306630/164568330-df0d5c08-3d2a-44ff-af94-2302c0b3611f.mov

Another user pointed out that all the built-in times use the format "Past N Time-Interval" but you dial your own custom time in as "N Time-Interval". Do we want to adjust this one way or the other to be uniform?

Another option would be to make it more rigid so people can't add in bogus values at all.

**Comment by azrikahar** [comment]
Moving this to discussion while we try to have a consensus on what would be the desired direction here 👍

Totally agree that it can be confusing. Maybe one simpler alternative is to make it so `Past` is prefixed and _stays_ there, if that makes sense.

Another slightly bigger lift may be to make it akin to the current is between filter (which may also resolve the bogus value question since now it isn't user manual input), but we'll probably need to do the "Past X day/month/year" calculation behind the scene via something like [differenceInDays](https://date-fns.org/v2.28.0/docs/differenceInDays) from date-fns:

![chrome_SVvSI35OUj](https://user-images.githubusercontent.com/42867097/164602766-e4848fba-f155-4646-9a63-1f6c87012371.png)

**Reply by benhaynes** [reply]
I think we can either:

1. Adjust the "other" placeholder with an example syntax
And/or...
2. Add a link to the appropriate docs in the field's note text

**Comment by azrikahar** [comment]
On a related note, seems like #8701 is still present based on your chart there. 🤔 Bummer that this particular oddity is not as easily reproducible as usual.

**Reply by erondpowell** [reply]
Yeah, that came "out of the box" with every instance of demo data I used so far btw.

**Reply by benhaynes** [reply]
I saw it on my demo yesterday, too. Might need to check our demo and reopen of needed.

**Reply by erondpowell** [reply]
> On a related note, seems like #8701 is still present based on your chart there. 🤔 Bummer that this particular oddity is not as easily reproducible as usual.

@azrikahar You can't reproduce? You want me to invite you to the cloud instance that this happens on?

**Reply by azrikahar** [reply]
It's okay 👍 this is currently being tracked in #13341
